google-map.html

```diff
@@ -431,7 +441,7 @@
         return;
       }
       this._mutationObserver = new MutationObserver( this._updateMarkers.bind(this));
-      this._mutationObserver.observe(this, {
+      this._mutationObserver.observe(this.$.selector, {
```

Comment:
When using
`this._mutationObserver.observe(Polymer.dom(this.$.selector), ...` or
`this._mutationObserver.observe(Polymer.dom(this), ...` I get the error:
`Failed to execute 'observe' on 'MutationObserver': The provided node was null.`.

When I change the code to
`this._mutationObserver.observe(Polymer.dom(this.$.selector).node, ...` (not sure if that makes sense) the test case still fails with native shadow dom enabled. If I change it to `this._mutationObserver.observe(Polymer.dom(this).node, ...` the native tests succeed, but the test case that previously failed with native shadow dom enabled now fails in shady dom.

Any ideas how to fix this?